addons/sourcemod/scripting/freak_fortress_2.sp

```diff
@@ -4565,7 +4568,7 @@ public OnClientDisconnect(client)
 	FF2flags[client]=0;
 	Damage[client]=0;
 	uberTarget[client]=-1;
-	playBGM[client]=true;
+	playBGM[client]=false; // Better set it false
```

Comment:
I think I see what you're saying now.  If `StopMusic(permanent)` was called globally then it looks like clients joining after that will still get music played the next round.  And now that `playBGM[client]` is set properly when the client joins this can be set to `false` to fix that.